ros/lib/models.py

```diff
@@ -23,7 +23,7 @@ class PerformanceProfile(db.Model):
     @property
     def display_performance_score(self):
         display_performance_score = {}
-        for key, value in self.performance_score.items():
+        for key, value in self.performance_utilization.items():
             score = value // 20
             display_performance_score[key] = 1 if score < 1 else score
 
```

Comment:
Let's rename `display_performance_score` to something like `display_performance_utilization` term?

Reply:
One thing about `display_performance_score` field is that it contains 1 to 5 numbers for keys (cpu/io/memory). Do you still want to rename this to utilization?

Reply:
gotcha, since we are keeping the score.